### ISSUE TITLE
Use different symbols to represent "open in container" and "data-updated" in pinned tabs

### ISSUE THREAD

**Issue by benplumley** [mbnuqw/sidebery#190]
## Description
The same symbol (a blue circle) is currently used for both "Open in [blue] container" and "Unread" when a tab is pinned. For example:

![firefox_2020-04-03_09-34-51](https://user-images.githubusercontent.com/7894368/78353482-05c26880-75a2-11ea-9105-eabd67d7e101.png)

My suggestion is that the symbol for "Open in container" be changed to a horizontal rectangle along the top or bottom of the pinned tab's box, similar to the vertical rectangle currently used to indicate containers for non-pinned tabs.

**Comment by aletaziar** [comment]
You mean "open in [blue] container" and "data-updated". Currently the pins don't show an an unread indicator, which is a new feature that just landed for normal tabs in 4.7.0.

I use the following code in the Styles Editor in Sidebery options to move the container indicator to a strip at the bottom of the pin. It looks like this:

![Annotation 2020-04-03 234219](https://user-images.githubusercontent.com/692914/78418739-c2e5ac80-7604-11ea-941a-35e529d013bc.png)

```
/* -------------- Move container indicator to bottom of pins */
.PinnedTab .ctx {
    --pin-container: 2px;
}
.PinnedTab .ctx {
    top: calc(100% - var(--pin-container));
    left: 0px;
    width: 100%;
    height: var(--pin-container);
    box-shadow: unset;
}
```

**Comment by mbnuqw** [comment]
:thinking: Actually I can mimic the native Firefox UI:

![screenshot-2020-04-06 13:33:06](https://user-images.githubusercontent.com/6276694/78539233-a9628180-780b-11ea-8b82-07aae9474a0b.png)
![screenshot-2020-04-06 13:35:25](https://user-images.githubusercontent.com/6276694/78539242-ac5d7200-780b-11ea-959e-2f556f1428bb.png)

p.s. I used circles as the easiest option for all kinds of pinned tabs layout (vertical - horizontal).

**Comment by mbnuqw** [comment]
Fixed in v4.9.0
